python/cudf/cudf/utils/applyutils.py

```diff
@@ -177,19 +188,24 @@ def compile(self, func, argnames, extra_argnames):
         )
         return kernel
 
-    def launch_kernel(self, df, args, chunks, tpb):
+    def launch_kernel(self, df, args, chunks, blkct=None, tpb=None):
```

Comment:
Would be nice to inject these new configuration into an existing apply-kernel test (or duplicate one)

Reply:
Done